core/include/thread_config.h

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2014-21 Freie Universität Berlin
+ *               2021 Inria
+ *               2021 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     core_thread
+ *
+ * @{
+ *
+ * @file
+ * @brief       Thread configuration defines
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef THREAD_CONFIG_H
+#define THREAD_CONFIG_H
+
```

Comment:
The CI wants the usual C++ compatibility boilerplate.

Reply:
fixed!